bucketeer/src/main/java/jp/bucketeer/sdk/ApiClient.kt

```diff
@@ -20,10 +19,8 @@ internal class ApiClient(
   private val client: GatewayGrpc.GatewayBlockingStub
 
   init {
-    val sslContext = getSSLContext()
     val channel = OkHttpChannelBuilder.forAddress(endpoint, getEndpointPort(endpoint))
         .useTransportSecurity()
-        .sslSocketFactory(sslContext.socketFactory)
```

Comment:
I am removing this because TLS 1.2 is enabled by default from API 21.
Currently, our `minSdkVersion` is set to 21. So this is no longer necessary. 

https://developer.android.com/reference/javax/net/ssl/SSLSocket#default-configuration-for-different-android-versions